server/@types/dto/index.d.ts

```diff
@@ -48,24 +48,24 @@ declare module 'dto' {
   export interface EducationDto extends ReferencedAndAuditable {
     prisonNumber: string
     educationLevel: EducationLevelValue
-    qualifications: Array<QualificationDto>
+    qualifications: Array<AchievedQualificationDto>
   }
 
-  export interface QualificationDto {
-    reference: string
+  export interface AchievedQualificationDto {
```

Comment:
Moved this up out of module `inductionDto` and use it in place of `QualificationDto` that I added earlier (as they are basically the same thing)